service/src/main/java/bio/terra/pipelines/app/controller/PipelinesApiController.java

```diff
@@ -34,11 +36,13 @@ public class PipelinesApiController implements PipelinesApi {
   @Autowired
   public PipelinesApiController(
       SamConfiguration samConfiguration,
+      PipelineConfigurations pipelineConfigurations,
       SamUserFactory samUserFactory,
       HttpServletRequest request,
       PipelinesService pipelinesService,
       QuotasService quotasService) {
     this.samConfiguration = samConfiguration;
+    this.pipelineConfigurations = pipelineConfigurations;
```

Comment:
can you show how this would be incorporated into the other places that use these configs, e.g. PipelineRuns?

Reply:
this is how i would imagine we use it https://github.com/DataBiosphere/terra-scientific-pipelines-service/pull/269/files#diff-05ecc06b443a680ff6163df9d0a1c525ee730bedef721eee2f32bc1276e88682R72